src/openms/source/ANALYSIS/OPENSWATH/PeakIntegrator.cpp

```diff
@@ -323,35 +323,33 @@ namespace OpenMS
   }
 
   template <typename PeakContainerConstIteratorT>
-  double PeakIntegrator::findPosAtPerc_(
+  double PeakIntegrator::findPosAtPeakHeightPercent_(
     PeakContainerConstIteratorT it_begin,
     PeakContainerConstIteratorT it_end,
     const double peak_height,
-    const double perc,
+    const double percent,
     const bool is_left_half
   ) const
   {
-    const double goal_intensity = peak_height * perc;
+    const double percent_intensity = peak_height * percent;
     PeakContainerConstIteratorT closest = is_left_half ? it_begin : it_end - 1; // TODO assuming enough points here
     if (is_left_half)
     {
-      for (PeakContainerConstIteratorT it = it_begin; it != it_end; ++it)
-      {
-        if (it->getIntensity() <= goal_intensity)
-        {
-          closest = it;
-        }
-      }
+      for (
```

Comment:
This is pretty cool.   Not only does it save a line, but it also breaks the loop when original if statement is no longer met.